builder/build.go

```diff
@@ -15,6 +15,7 @@ import (
 	v1execute "github.com/alexellis/go-execute/pkg/v1"
 	"github.com/openfaas/faas-cli/schema"
 	"github.com/openfaas/faas-cli/stack"
+	"github.com/openfaas/faas-cli/utils"
```

Comment:
I can't see why a separate package is needed. I would suggest removing that package and putting the code where it is consumed.

Packages named utils are almost always a code smell.

Reply:
Resolved